react/Field/index.jsx

```diff
@@ -133,6 +134,17 @@ const Field = props => {
   return (
     <div className={cx(styles['o-field'], className)}>
       <Label htmlFor={id}>{label}</Label>
+      {sideLabel && (
+        <div
+          className={cx(
+            styles['o-side-label'],
+            LabelStyles['c-label'],
+            styles['o-field-input-action']
```

Comment:
It think that we should keep semantic here, as we are not displaying an _input action_ element, as we do for the password, we should not use a class named like that.

Instead, I propose to use `o-field-input-action` CSS properties into `o-side-label`, with a mixin or something.

What @GoOz says about this ?